test/test_designator/test_location_designator.py

```diff
@@ -10,6 +12,8 @@ def test_reachability(self):
         self.robot.set_joint_position(RobotDescription.current_robot_description.torso_joint, 0.3)
         object_desig = ObjectDesignatorDescription(names=["milk"])
         robot_desig = ObjectDesignatorDescription(names=[RobotDescription.current_robot_description.name])
+        with simulated_robot:
+            action_designator.ParkArmsAction([Arms.BOTH]).resolve().perform()
```

Comment:
Are these really necessary? I would like to keep the different designators apart in the tests, otherwise the result of the tests is now also depended of the ParkArmsAction even though that is not the target of the test.